packages/story-editor/src/app/media/utils/useMediaUploadQueue/reducer.js

```diff
@@ -139,6 +141,7 @@ export function finishUploading(state, { payload: { id, resource } }) {
         ? {
             ...item,
             resource,
+            previousResourceId: item.resource.id,
```

Comment:
What if resource is null?

Reply:
No issue. Then `useUploadMedia` simply won't use it.

Reply:
But then you are trying to access an object that doesn't exist?

Reply:
Misread; here, `resource` is never `null` as we're finishing uploading a resource. It's always an object at this point.